backend/apps/accounts/views.py

```
@@ -600,625 +120,46 @@ def post(self, request):
         )
 
 
-# ─────────────────────────────────────────────────────────────────────────────
-# OTP (Email Verification) Views
-# ─────────────────────────────────────────────────────────────────────────────
-
-
 @extend_schema(
```

Ruff (0.15.20)

[error] Undefined name `extend_schema`

(F821)